platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -53,9 +53,7 @@
 import com.swirlds.common.metrics.platform.DefaultMetricsProvider;
 import com.swirlds.common.startup.CommandLineArgs;
 import com.swirlds.common.startup.Log4jSetup;
-import com.swirlds.common.system.NodeId;
-import com.swirlds.common.system.SoftwareVersion;
-import com.swirlds.common.system.SwirldMain;
+import com.swirlds.common.system.*;
```

Comment:
I assume we do not want to have *-Imports

Reply:
I never look at the import statements, so I don't know what they contain. I thought spotless takes care of that as well. I don't really have a strong opinion on it. Is there a policy for this?

Reply:
I did a merge from develop and these imports have been removed